cpp/include/Ice/LocalException.h

```diff
@@ -3569,45 +3569,6 @@ namespace Ice
          */
         ICE_MEMBER(ICE_API) virtual void ice_print(::std::ostream& stream) const override;
     };
-
-    /**
-     * Indicates the application attempted to call the callbacks for an async dispatch more than once, or called an
-     * async dispatch callback after throwing an exception from the dispatch thread. \headerfile Ice/Ice.h
-     */
-    class ICE_CLASS(ICE_API) ResponseSentException : public LocalExceptionHelper<ResponseSentException, LocalException>
```

Comment:
Removed this exception. If you attempt to call twice an AMD callback, the second call will be no-op. I want the AMD callbacks to be logically noexcept, just like sendResponse.